README.md

```diff
@@ -62,6 +56,17 @@ See the [CONTRIBUTING](https://github.com/pytorch/opacus/tree/main/CONTRIBUTING.
 
 Do also check out our README files inside the repo to learn how the code is organized.
 
+## Citation
+To cite Opacus in your papers (much appreciated!), please use the following:
+```
+@article{opacus,
+  title={Opacus: User-Friendly Differential Privacy Library in PyTorch},
+  author={A. Yousefpour and I. Shilov and A. Sablayrolles and D. Testuggine and K. Prasad and M. Malek and J. Nguyen and S. Ghosh and A. Bharadwaj and J. Zhao and G. Cormode and I. Mironov},
+  journal={arXiv preprint arXiv:2109.12298},
+  year={2021}
+}
+```
+
 ## References
```

Comment:
let's remove these. We are not writing a paper to have references. This is a library and sure it uses a lot of resource (and not only just these)